apps/dash-daq-satellite-dashboard/data/gps_data_h.csv

```diff
@@ -0,0 +1,61 @@
+lat,lon
```

Comment:
Why is this file full of zeroes?

Reply:
Need if for default values

Reply:
Need it for default values

Reply:
Could you instead generate this from within the Python file?